tensorboard/plugins/hparams/tf_hparams_scale_and_color_controls/tf-hparams-scale-and-color-controls.html

```diff
@@ -251,19 +251,17 @@
         if (!this._isNumericColumn(column.index)) {
           return false;
         }
-        const [min, max] = tf.hparams.utils.numericColumnExtent(
+        const [min, max] = tf.hparams.utils.visibleNumericColumnExtent(
             this.schema, this.sessionGroups, column.index);
         // Log scale is only defined when the domain does not include 0.
         return min > 0 || max < 0;
       },
 
-      _notifyOptionsChanged() {
-        // We rewrite options here to notify polymer that 'options' have
-        // changed so that other elements that depend on options will be
-        // updated.
-        const options = this.options;
-        this.options = {};
-        this.options = options;
+      _scaleChanged(event) {
+        // Notify polymer that the column changed so that other elements
+        // that depend on it will see the change.
+        const model = event.model;
+        this.notifyPath(`options.columns.${model.column.index}.scale`);
```

Comment:
Wow, I am very surprised how things are not working properly. I have tried to figure out what is going on underneath to learn quirks about Polymer I may not know about. I tried breaking `options` into smaller private props and tried to make the notifyPath smaller but that didn't help at all. It is super unfortunate that this has to happen but since we, collectively, cannot figure out why, let's just let this be.

Reply:
Ok. Thanks for trying to find a better alternative.